index.html

```diff
@@ -0,0 +1,162 @@
+      </div>
+      <div class="button-contaier">
+        <button class="button-campaign">Join C&C</button>
+      </div>
+    </nav>
+    <i class="fa-solid fa-bars mobile-menu mobile_element black"></i>
+  </header>
+  <main>
+    <section class="introduction">
+      <span class="introduction-hello">Console.log("hello world!")</span> <br>
+      <h1> COFFEE AND CODE<br> ANNUAL MEETING<br> 2022</h1>
+    </section>
+    <section class="introduction-detail black">
+      <p> A joyful celebration believing in the value of openness and sharing, creating a poseitive change with people
+        from over 80 countries is taking place in October, in Ethiopia.</p>
+    </section>
+    <section class="event-place-time black">
+      <div class="event-date black">2022.10.15(THU) ~ 16 (FRI)</div>
+      <div class="event-place black"> <span class="place-at">&#64;</span> SkyLight hottel, Addis Ababa, Ethiopia</div>
+
+    </section>
+    <section class="main-program">
+      <header class="white lato main-program-title"> Main Program</header>
+      <div class="programs">
+        <div class="program">
+
+          <div class="program-title orange">
+            <i class="fa fa-book icons" aria-hidden="true"></i>
+            <p>Lecture</p>
+          </div>
+          <div class="program-detail white lato">Listen to the speaker from various countries about the messeges of
+            sharing
+            and
+            <p>opening</p>
+          </div>
+        </div>
+        <div class="program">
+          <div class="program-title orange">
+            <i class="fa fa-book icons" aria-hidden="true"></i>
+            <p>CC Exhibition</p>
+          </div>
+          <div class="program-detail white">Appreciate various creations applying CC license of artists, organized from
+            <p>Art
+              Center Nabi.
+            </p>
+          </div>
+        </div>
+        <div class="program">
+          <div class="program-title orange">
+            <i class="fa fa-book icons" aria-hidden="true"></i>
+            <p>Forum</p>
+          </div>
+          <div class="program-detail white">Have the time to share your thoughts and opinions with experts for each
+            topic
+          </div>
+        </div>
+        <div class="program">
+          <div class="program-title orange">
+            <i class="fa fa-book icons" aria-hidden="true"></i>
+            <p>Workshop</p>
+          </div>
+          <div class="program-detail white">Try creating your own work using open source license rather than just
+            watching
+          </div>
+        </div>
+        <div class="program">
+          <div class="program-title orange">
+            <i class="fa fa-book icons" aria-hidden="true"></i>
+            <p> CC Ignite</p>
+          </div>
+          <div class="program-detail white">Get opportunities to network with CC affiliates around the worl, also after
+            the summit</div>
+        </div>
+      </div>
```

Comment:
- [ ] Kindly use the correct font weight and sides as it is shown in the Behance design.

**This is how your design currently looks like:**

![image](https://user-images.githubusercontent.com/86318284/181342430-d40c5180-e5c3-4fdf-9ec8-3276502ab79b.png)

**This is the desired Behance design:**

![image](https://user-images.githubusercontent.com/86318284/181342602-0ca103f0-9e55-4b32-99ea-5b23bab5b8d6.png)